internal/pkg/bulk/bulk_integration_test.go

```diff
@@ -69,8 +69,8 @@ func TestBulkCreate(t *testing.T) {
 			Name:  "Invalid utf-8",
 			Index: string([]byte{0xfe, 0xfe, 0xff, 0xff}),
 			Err: es.ErrElastic{
-				Status: 400,
-				Type:   "parse_exception",
+				Status: 500,
+				Type:   "json_parse_exception",
```

Comment:
Revert changes made in https://github.com/elastic/fleet-server/pull/3931, it looks like ES reverted the change that was in the snapshot